### PR TITLE
Timestamp

### PR DESCRIPTION
**Pull request by klauswuestefeld**

### CODE REVIEW

File: biz/timestamp/project.clj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+(defproject house.jux/biz.timestamp "2024.08.30-SNAPSHOT"
```
Comment (klauswuestefeld) [inline biz/timestamp/project.clj]: Try using nil instead of ::no-result in result project.

File: test/spread/src/house/jux__/test/spread__.clj [hunk 2]
```diff
@@ -369,7 +370,8 @@
 (defn- run-tests-in-namespace! [namespace-folder]
   (let [subject-namespace (-> namespace-folder .getName symbol)
         empty-context     {:state {}}]
-    (run-tests-in-folder! empty-context subject-namespace namespace-folder)))
+    (binding [*timestamp* (reset-timestamp)]
```
Comment (klauswuestefeld) [inline test/spread/src/house/jux__/test/spread__.clj]: Remove

File: biz/timestamp/project.clj (hunk 1 above)
Comment (kauwai) [reply biz/timestamp/project.clj]: Solved in 5cad3fd

File: test/spread/src/house/jux__/test/spread__.clj (hunk 2 above)
Comment (kauwai) [reply test/spread/src/house/jux__/test/spread__.clj]: Solved in b361d5d623ea5f446a50ee65d926b4571a4fa0f3

**Pull request merged**